docs/reference/commandline/index.md

```diff
@@ -62,20 +62,20 @@ read the [`dockerd`](dockerd.md) reference page.
 | [events](events.md) | Get real time events from the server                   |
 | [exec](exec.md) | Run a command in a running container                       |
 | [export](export.md) | Export a container's filesystem as a tar archive       |
-| [kill](kill.md) | Kill a running container                                   |
+| [kill](kill.md) | Kill one or more running containers                        |
```

Comment:
I'm still in doubt if `one or more` makes it more readable in this overview (or in general). (I know it's the description that's currently display in the usage), but not sure what would be best to show here.

Maybe keep the old description for the overview, and leave the "one or more" to the detail page.